src/index.js

```diff
@@ -71,18 +70,23 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
   }
 
   logger.info('The scheduler is running…');
-  logger.info('Documents will be tracked at minute 30 past every 6 hours.');
-  scheduler.scheduleJob('30 */6 * * *', async () => {
+
+  const TRACK_CHANGES_HOURS_INTERVAL = 2;
```

Comment:
Currently, we generate too much noise on each tracking run. Let's decrease this until we make it configurable.
```suggestion
  const TRACK_CHANGES_HOURS_INTERVAL = 24;
```